xcmailr-webapp/src/main/java/services/CheckDBForMailAddressDuplicates.java

```diff
@@ -38,8 +38,8 @@ private void validateUserAddresses() throws Throwable
     {
         final HashSet<String> mailAddresses = new HashSet<>();
         final HashMap<Long, String> idToMailAddress = new HashMap<>();
-        final PagingList<User> users = Ebean.find(User.class).select("id, mail").findPagingList(1_000);
-        for (final User user : users.getAsList())
+        final PagedList<User> users = Ebean.find(User.class).select("id, mail").setMaxRows(1_000).findPagedList();
+        for (final User user : users.getList())
```

Comment:
Please check if this construct really finds and processes all users or just the first 1,000.

Reply:
Changed to iterator approach (recommended for queries w/ large result sets).